cmd/gen-go-sample/main.go

```diff
@@ -111,7 +115,13 @@ func genMethodSamples(gen *generator, iface GAPICInterface, meth GAPICMethod, no
 			if err := gen.genSample(iface.Name, meth.Name, sam.RegionTag, vs); err != nil {
 				return err
 			}
-			if err := gen.commit(!nofmt, time.Now().Year(), os.Stdout); err != nil {
+			content, err := gen.commit(!nofmt, time.Now().Year())
+			if err != nil {
+				return err
+			}
+
+			fname := iface.Name + "_" + meth.Name + "_" + vsID + ".go"
+			if err := ioutil.WriteFile(filepath.Join(outDir, fname), content, 0644); err != nil {
```

Comment:
YAY! 🎉 – you didn't call this out in the PR description/title

Reply:
Updated :)